dynamo/preprocessing/preprocess.py

```diff
@@ -1238,6 +1242,7 @@ def select_genes(
 def recipe_monocle(
     adata: anndata.AnnData,
     reset_X: bool = False,
+    feature_selection_layer: Union[list, np.ndarray, np.array] = None,
```

Comment:
MAKE SURE this argument won't affect the old behavior and only used for selecting feature genes but nothing else.